apps/web/src/actions/datasets/create.ts

```diff
@@ -6,12 +6,6 @@ import { z } from 'zod'
 
 import { authProcedure } from '../procedures'
 
-const ACCEPTED_FILE_TYPES = [
-  'text/csv',
-  'application/vnd.ms-excel',
-  'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
-  'application/vnd.oasis.opendocument.spreadsheet',
-]
 const MAX_SIZE = 3
```

Comment:
3mb only? that's very small

Reply:
25mb of text?

Reply:
well the theory is that they want to evaluate on a large body of data no?, 3mb doesn't push our infra hard at all and it can be limiting for no reason

Reply:
25 then. But node server will handle it? 

We'll have to do this https://nextjs.org/docs/app/api-reference/next-config-js/serverActions#bodysizelimit

Although the best approach would be to direct upload to S3 and then process the file when S3 responds. But we need to change how upload is done

Reply:
I'll start with 15MB and see if someone complains